apps/client/src/app/implementation-guide-wrapper/r4/implementation-guide.component.html

```diff
@@ -568,6 +555,26 @@ <h2>
       </ng-template>
     </ngb-tab>
 
+    <!-- Publishing -->
+    <ngb-tab id="publishing" title="Publishing">
+      <ng-template ngbTabContent>
+        <div class="row">
+          <div class="col-md-12">
+            <trifolia-fhir-package-list
+              [implementationGuide]="implementationGuide"
+              [defaultPackageId]="packageId"
+              [defaultTitle]="this.implementationGuide.title"
+              [defaultName]="this.implementationGuide.name" (change)="isDirty = true; nameChanged();">
+            </trifolia-fhir-package-list>
+          </div>
+
+          <div class="col-md-12">
+            <trifolia-fhir-ignore-warnings [implementationGuide]="implementationGuide" (change)="isDirty = true"></trifolia-fhir-ignore-warnings>
```

Comment:
Need to add in "nameChanged()" to the (change) attribute here.  Makes sure that changes register on the browser tab name and appropriately generate a * on the browser tab's name.

Reply:
Why is `nameChanged()` required? What does that do that `isDirty` doesn't? I don't think the ignoreWarnings.txt changes should affect the name of the IG. Maybe I'm missing something, though...